python/demo/demo_gmsh.py

```diff
@@ -120,7 +120,7 @@
 entities, values = distribute_entity_data(msh, 2, marked_facets, facet_values)
 
 msh.topology.create_connectivity(2, 0)
-mt = meshtags_from_entities(msh, 2, create_adjacencylist(entities), np.int32(values))
+mt = meshtags_from_entities(msh, 2, create_adjacencylist(entities), np.array(values, dtype=np.int32))
```

Comment:
Why doesn't `values` have the correct type in the first place?

Reply:
mypy cannot infer the return types of functions such as `distribute_entity_data` that are exposed directly from pybind11, e.g.:

`demo_gmsh.py:121: note: Revealed type is "Any"` for `values`.

The only solution is to lightly wrap them in Python and specify the types (again). What do you think?

Reply:
I wrapped the method with the types.

There do seem to be projects that claim to automagically create typing data for binary modules, but we only have a few methods to wrap. Seems like overkill.

Reply:
Fixed.